qiime2/core/cache.py

```diff
@@ -567,11 +575,39 @@ def is_cache(cls, path):
 
         regex = \
             re.compile(
-                r"QIIME 2\ncache: \d+\nframework: 20\d\d\.")
+                r"QIIME 2\ncache: v?\d+\nframework: 20\d\d\.")
```

Comment:
```suggestion
                r"QIIME 2\ncache: v?(\d+)\nframework: (20\d\d\.\d+)")
```
If there was a capture group we could skip the yaml parsing and fh schnaedling (Liz says this is a word).

Reply:
So. I am adding a regex (gross) with reasoning based on a fake word (gross gross)